app/components/Home.js

```diff
@@ -0,0 +1,675 @@
+var React = require('react');
+var styles = require('../styles');
+var ReactRouter = require('react-router');
+var Link = ReactRouter.Link;
+
+var Home = React.createClass({
+
+    getInitialState: function () {
+        return {
+            drinkList: [
```

Comment:
Might want to put this in another file and import it so it doesn't clutter you Home component